recipe/meta.yaml

```diff
@@ -23,7 +23,6 @@ requirements:
     - setuptools
   run:
     - python
-    - six
     - sqlalchemy >=1.3
     # extras
     - babel >=1.3
```

Comment:
Although this is not necessarily a blocker, I think `importlib_metadata` should be moved above the `# extras` since it is required.